src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -317,6 +317,62 @@ public Invocation TransformInvocationVersionOne(InvocationVersionOne node)
             return invocation;
         }
 
+        public Notification TransformNotificationVersionOne(NotificationVersionOne node)
+        {
+            Notification notification = null;
+
+            if (node != null)
+            {
+                notification = new Notification
+                {
+                    Exception = TransformExceptionDataVersionOne(node.Exception),
+                    Id = node.Id,
+                    Level = node.Level.ToNotificationLevelV2(),
```

Comment:
Y'know, since all the other transformations are done by methods with names `TransformXxxVersionOne`, perhaps this should be
```
Level = TransformNotificationLevelVersionOne(node.Level);
```

Reply:
OK